static/app/views/replays/replayTable/index.tsx

```diff
@@ -69,6 +72,40 @@ function ReplayTable({
     .filter(Boolean)
     .map(column => <HeaderCell key={column} column={column} sort={sort} />);
 
+  const {
+    selection: {projects},
+  } = usePageFilters();
+
+  const MIN_DEAD_RAGE_CLICK_SDK = '7.60.1';
+
+  const needSDKUpgrade = useProjectSdkNeedsUpdate({
+    minVersion: MIN_DEAD_RAGE_CLICK_SDK,
+    organization,
+    projectId: projects.map(String),
+  });
+
+  if (needSDKUpgrade.needsUpdate) {
+    return (
+      <StyledPanelTable
+        headers={tableHeaders}
+        isLoading={false}
+        visibleColumns={visibleColumns}
+        data-test-id="replay-table"
+        gridRows={undefined}
+      >
+        <StyledAlert type="info" showIcon>
+          {tct('[data] require an [sdkPrompt]. [link:Learn how to upgrade.]', {
```

Comment:
I would consider a shorter CTA so that it fits on a single line too

```suggestion
          {tct('[data] require an [sdkPrompt]. [link: Upgrade now.]', {
```

or

```suggestion
          {tct('[data] require an [sdkPrompt]. [link: Upgrade]', {
```